File: android/app/src/main/java/com/lightningkite/kwifttemplate/shared/api/APIInterface.kt
package com.lightningkite.kwifttemplate.shared.api

import com.lightningkite.kwift.actual.escaping
import com.lightningkite.kwifttemplate.shared.models.Post

interface APIInterface {
    //https://jsonplaceholder.typicode.com/posts
    fun getExamplePosts(
        onResult: @escaping() (Int, List<Post>?, String?) -> Unit
    )
}

File: ios/project/shared/api/APIInterface.swift
import Foundation
//package com.lightningkite.kwifttemplate.shared.api


//import com.lightningkite.kwift.actuals.Empty
//import com.lightningkite.kwift.actuals.ImageData
//import com.lightningkite.kwift.actuals.escaping
//import com.lightningkite.kwifttemplate.shared.models.*

public protocol APIInterface {
    //https://jsonplaceholder.typicode.com/posts
    
    
    func getExamplePosts(
        onResult: @escaping (Int, [Post]?, String?) -> Void
    )
}
